src/serializer/MapFishPrintV3XYZSerializer.ts

```diff
@@ -0,0 +1,29 @@
+import OlLayer from 'ol/layer/Layer';
+import OlSourceXYZ from 'ol/source/XYZ';
+import OlSource from 'ol/source/Source';
+import MapFishPrintV3OSMSerializer from './MapFishPrintV3OSMSerializer';
+
+export class MapFishPrintV3XYZSerializer extends MapFishPrintV3OSMSerializer {
```

Comment:
Semantically this is misleading. I would favour that the `MapFishPrintV3OSMSerializer` extends the `MapFishPrintV3XYZSerializer` as the OSM layer is a subclass of the XYZ layer.

Reply:
resolved in [bb9a6cb](https://github.com/terrestris/mapfish-print-manager/pull/482/commits/bb9a6cb71f831a645d52eff4b1b1fa7c27cda377) and depends on [ecf2d9b](https://github.com/terrestris/shogun-gis-client/pull/1531/commits/ecf2d9b4232e210b202dab90885703b2dfcfefad)